components/GlobalStyle/GlobalStyle.js

```diff
@@ -5,8 +5,9 @@ export const GlobalStyle = createGlobalStyle`
     box-sizing: border-box;
     margin:0;
     padding:0;
+    
+    
   }
   body {
-    
```

Comment:
I would recommend to delete the body style as long as it is empty

Reply:
you're right! Thanks!